com.cubrid.common.ui/src/com/cubrid/common/ui/cubrid/table/export/handler/AbsExportDataHandler.java

```diff
@@ -202,22 +203,41 @@ protected void setIsHasBigValue(String columnType, int precision) { // FIXME mov
 
 	protected String getSelectSQL(String name) { // FIXME move this logic to core module
 		String sql = null;
-		if (exportConfig.getSQL(name) != null) {
-			sql = exportConfig.getSQL(name);
-		} else {
-			List<String> columnList = exportConfig.getColumnNameList(name);
-			StringBuffer columns = new StringBuffer();
-			int size = columnList.size();
-			for (int i = 0; i < columnList.size(); i++) {
-				columns.append(QuerySyntax.escapeKeyword(columnList.get(i)));
-				if (i != size - 1) {
-					columns.append(',');
-				}
+		List<String> columnList = getColumnList(name);
+		StringBuffer columns = new StringBuffer();
```

Comment:
@seunghun-kim Would you replace the `StringBuffer` by `StringBuilder` due to unnecessary thread safe class.